lms/djangoapps/certificates/signals.py

```diff
@@ -93,5 +95,6 @@ def fire_ungenerated_certificate_task(user, course_key):
         generate_certificate.apply_async(kwargs={
             'student': unicode(user.id),
             'course_key': unicode(course_key),
+            'verification_status': unicode(verification_status)
```

Comment:
```python
'expected_verification_status': unicode(expected_verification_status) if expected_verification_status else None
```
or you can also do this type of thing:
```python
kwargs = {
    'student': 'blah',
    'course_key': 'blah'
}
if expected_verification_status:
    kwargs['expected_verification_status'] = unicode(expected_verification_status)
generate_certificate.apply_async(kwargs=kwargs)

Reply:
Wouldn't the top one send an argument "expected_verification_status" of None? And the bottom one will only pass in "expected_verification_status" if it exists?

Reply:
Yes.  The way you have the task setup, either is fine, so it's really just a matter of taste.